lib/oauth2.js

```diff
@@ -84,74 +65,24 @@ exports.OAuth2.prototype._request= function(method, url, headers, post_body, acc
     realHeaders['User-Agent'] = 'Node-oauth';
   }
 
-  if( post_body ) {
```

Comment:
request sets the `content-length` for us: https://github.com/request/request/blob/2d95c753f6005b914f544364a0543e13c6656f78/request.js#L618